src/pages/MockMan/MockMan.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import Mockman from "mockman-js";
+function MockAPI() {
+  return (
+    <div className="MockAPI">
```

Comment:
No need to create this extra file for mockman i guess you can directly 
import Mockman from "mockman-js"; & use as <Route path="/mockman" element={<Mockman />} />
where it is requird

Reply:
Thanks, will try that for next project